internal/pkg/earpointer/taint.go

```diff
@@ -153,31 +155,34 @@ func (ht *heapTraversal) canReach(sink ssa.Instruction, sources []*source.Source
 	return nil
 }
 
-// For a function, transitively get the functions called within this function.
-// Argument "depth" controls the depth of the call chain.
-// For example, return {g1,g2,g3} for "func f(){ g1(); g2() }, func g1(){ g3() }".
-func calleeFunctions(fn *ssa.Function, result map[*ssa.Function]bool, depth uint) {
-	if depth <= 0 {
+// For a function, transitively get the functions reachable from this function
+// according to the call graph. Both callers and callees are considered.
+// Argument "depth" controls the depth of the call chain. For example,
+//   func f(){ g1(); g2() }
+//   func g1(){ g3() }
+// for input "g1", return {f,g1,g2,g3} if depth>1, and return {f, g1} if depth=1.
+func reachableFunctions(fn *ssa.Function, result map[*ssa.Function]bool, cg *callgraph.Graph, depth uint) {
```

Comment:
result is an output parameter. This probably should be the last or first argument and documented.